.travis.yml

```diff
@@ -1,32 +1,26 @@
-# while this doesn't require sudo we don't want to run within a Docker container
-sudo: true
-dist: trusty
-language: python
-python:
-  - "3.4"
+# This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci package.
+sudo: required
+dist: xenial  # distro used by Travis, moveit_ci uses the docker image's distro
+services:
+  - docker
+language: cpp
+compiler: gcc
+cache: ccache
+
 env:
   global:
-    - JOB_PATH=/tmp/devel_job
+    - DOCKER_IMAGE=moveit/moveit:master-source
+    - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-function"
+    - WARNINGS_OK=false
   matrix:
-    - ROS_DISTRO_NAME=kinetic OS_NAME=ubuntu OS_CODE_NAME=xenial ARCH=amd64
-install:
-  # either install the latest released version of ros_buildfarm
-  - pip install ros_buildfarm
-  # checkout catkin for catkin_test_results script
-  - git clone https://github.com/ros/catkin /tmp/catkin
-  # run devel job for a ROS repository with the same name as this repo
-  - export REPOSITORY_NAME=`basename $TRAVIS_BUILD_DIR`
-  # use the code already checked out by Travis
-  - mkdir -p $JOB_PATH/catkin_workspace/src
-  - cp -R $TRAVIS_BUILD_DIR $JOB_PATH/catkin_workspace/src/
-  # generate the script to run a devel job for that target and repo
-  - generate_devel_script.py https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml $ROS_DISTRO_NAME default $REPOSITORY_NAME $OS_NAME $OS_CODE_NAME $ARCH > $JOB_PATH/devel_job.sh
-  - cd $JOB_PATH
-  - cat devel_job.sh
-  # run the actual job which involves Docker
-  - sh devel_job.sh -y
+    - TEST="clang-format catkin_lint"
+    - TEST=clang-tidy-fix
+    - DOCKER_IMAGE=moveit/moveit:melodic-source
+    - DOCKER_IMAGE=moveit/moveit:master-source
+    - DOCKER_IMAGE=moveit/moveit:kinetic-ci  UPSTREAM_WORKSPACE=moveit.rosinstall  TEST_BLACKLIST=moveit_ros_perception
```

Comment:
I think this should  use `rviz_visual_tools.rosinstall` which doesn't exist yet

Reply:
```suggestion
    - DOCKER_IMAGE=moveit/moveit:kinetic-ci  TEST_BLACKLIST=moveit_ros_perception
```